bundles/org.openhab.voice.actiontemplatehli/src/main/java/org/openhab/voice/actiontemplatehli/internal/configuration/ActionTemplateConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.voice.actiontemplatehli.internal.configuration;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.items.Metadata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
I do not see these dependencies in your POM file. Are they inherited with NLP dependency ?
Is there no need to mention it in the NOTICE file ?

Reply:
I'm not sure.
I though those are available to all the project, because I see more bindings using those without any notice around.
But maybe those are in fact included by the library.

Reply:
No I think those are available to all the project because I don't see the opennlp-tools depending on those.
But not totally sure.

Reply:
I have the feeling it is included with the OH core framework:
https://github.com/openhab/openhab-core/blob/66953210898d4729dff08eee6dc72778cfeff28f/bom/runtime/pom.xml#L944
But this is not mentioned as default library in the binding development documentation: https://www.openhab.org/docs/developer/guidelines.html#default-libraries

Reply:
The chrome cast binding is doing this:
```
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-core</artifactId>
      <version>${jackson.version}</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-annotations</artifactId>
      <version>${jackson.version}</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-databind</artifactId>
      <version>${jackson.version}</version>
      <scope>compile</scope>
    </dependency>
```
Uses a global version variable. I think I can go for the same.

Reply:
@lolodomo I have added it as in the chromecast binding and in the notice file

Reply:
Yes and the pattern was the same in the iCalendar binding. This is probably the correct approach.